nixos/modules/services/networking/zerotierone.nix

```diff
@@ -29,6 +31,19 @@ in
 
   options.services.zerotierone.package = mkPackageOption pkgs "zerotierone" { };
 
+  options.services.zerotierone.localConf = mkOption {
+    default = { };
+    description = mdDoc ''
+      Configuration to be written to the zerotier JSON-based local.conf.
+      The configuration will be symlinked to /var/lib/zerotier-one/local.conf at build time.
+      To understand the configuration format, refer to https://docs.zerotier.com/config/#local-configuration-options.
+      '';
+    example = {
+      settings.allowTcpFallbackRelay = false;
+    };
+    type = types.attrs;
```

Comment:
Actually I think it would be better to make the default `null`, the type `nullOr attrs`, and don't touch the file in `/var/lib` unless `localConf` is not null. This way nothing changes unless the option is set.

Reply:
Good point. Let me see how I can change that.

Reply:
Done!

Reply:
Done!